python/pyspark/sql/functions.py

```diff
@@ -210,6 +210,44 @@ def sumDistinct(col):
     return _invoke_function_over_column("sumDistinct", col)
 
 
+@since(3.2)
```

Comment:
@rwpenney can you remove this and add `.. versionadded: 3.2.0` manually between the description and `Parameters`?

Reply:
Now changed as suggested. Thanks.